packages/core/app/styles/navi-core/components/navi-loader.less

```diff
@@ -15,7 +15,7 @@
     align-items: center;
     background-color: @navi-container-gray;
     display: flex;
-    flex: 1;
+    flex: 1 1 auto;
```

Comment:
We could just do `flex: auto` since this seems to be equivalent to that. 
https://css-tricks.com/almanac/properties/f/flex/

Reply:
I think `&__spinner` just be the same as 

https://github.com/yahoo/navi/blob/a62677653c9168e22258d0981a58f9f9cba10354/packages/core/app/styles/navi-core/common/spinner.less#L6-L13

(the results are the same)

Is there any way to directly reference that, or should I just make the attributes for `&__spinner` the same as `.spinner`

Reply:
You can just say 
`&__spinner {`
`.spinner;`
`}`